app.py

```diff
@@ -79,6 +80,16 @@ def refresh():
 
     return jsonify(f'The datasets updated successfully! The current number of available datasets is {len(masader)}.')
 
+@app.route('/cards/issue/<int:index>')
+def report_card_issue(index: int):
+
+    if not (1 <= index <= len(masader)):
+        return jsonify(f'Dataset index is out of range, the index should be between 1 and {len(masader)}.'), 404
+
+    title = request.args.get('title', default="", type=str)
+    message = request.args.get('message', default="", type=str)
+
+    report_issue(title, message);
```

Comment:
After creating the issue, the endpoint should return something to the frontend, like created successfully, or the link of the issue in JSON, right? Returning null will crash the request.